doc/source/user_guide/computation.rst

```diff
@@ -5,6 +5,53 @@
 Computational tools
 ===================
 
+.. _computation.ufuncs:
+
```

Comment:
should reference ``DataFrame interoperability with NumPy functions`` _dsintro.numpy_interop here, and/or remove the previous and reference this section.